scipy/stats/tests/test_distributions.py

```diff
@@ -2159,6 +2159,10 @@ def test_infinite_input():
     assert_almost_equal(stats.skellam.sf(np.inf, 10, 11), 0)
     assert_almost_equal(stats.ncx2._cdf(np.inf, 8, 0.1), 1)
 
+def test_gompertz_accuracy():
+    # Regression test for gh-4031
+    p = stats.gompertz.ppf(stats.gompertz.cdf(1e-100,1),1)
+    assert_equal(p, 1e-100)
```

Comment:
This should be ``assert_allclose(p, 1e-100)`` - comparing floating point numbers for exact equality usually leads to trouble.